app/models/manageiq/providers/openstack/cloud_manager/cloud_tenant.rb

```diff
@@ -41,13 +41,13 @@ def raw_delete_cloud_tenant
     raise MiqException::MiqCloudTenantDeleteError, e.to_s, e.backtrace
   end
 
-  private
-
   def self.connection_options
     connection_options = {:service => "Identity", :openstack_endpoint_type => 'adminURL'}
     connection_options
   end
 
+  private
```

Comment:
Sure, do you mean just revert the changes to this file? The commit made `connection_options` public, so reverting would make it private again.